lang/nqp/Portfile

```diff
@@ -43,3 +43,6 @@ universal_variant   no
 # configure.universal_args-delete --disable-dependency-tracking
 
 test.run            yes
+
+# we need to avoid '-' in version name, else $version-RC1 is considered newer than the actual release
+livecheck.regex     {archive/([0-9.]+).tar.gz}
```

Comment:
this should just be the usual `github.livecheck.regex  {([0-9.]+)}` and no need for a comment, this is quite common